test_scripts/API/ButtonSubscription/036_SubscribeButton_Custom_Button_send_event_to_Background.lua

```diff
@@ -0,0 +1,99 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0192-button_subscription_response_from_hmi.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) Mobile app is subscribed for CUSTOM_BUTTON Alert and Default action
+-- 2) App is sent to hmi level BACKGROUND
+-- 3) Mobile sends OnButtonEvent and OnButtonPress
+-- SDL does:
+-- 1) Respond OnButtonEvent and OnButtonPress to mobile app for CUSTOM_BUTTON ALERT
+-- 2) Not respond OnButtonEvent and OnButtonPress to mobile app for CUSTOM_BUTTON DEFAULT_ACTION
+---------------------------------------------------------------------------------------------------
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/ButtonSubscription/common_buttonSubscription')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local variable ]]
+local buttonName = "CUSTOM_BUTTON"
+local customButtonIDAlert = 1
+local alertId
+
+local RequestAlert =
+    {
+        alertText1 = "alertText1",
+        softButtons =
+        {
+          {
+            type = "BOTH",
+            text = "Close",
+            image =
+            {
+                value = "action.png",
+                imageType = "DYNAMIC",
+            },
+            isHighlighted = true,
+            softButtonID = customButtonIDAlert,
+            systemAction = "KEEP_CONTEXT",
+          }
+        }
+    }
+
+-- [[ Local Functions ]]
+local function pTUpdateFunc(pTbl)
+pTbl.policy_table.module_config.notifications_per_minute_by_priority["NONE"] = 2
+pTbl.policy_table.functional_groupings["Base-4"].rpcs["Alert"].hmi_levels = {
+    "FULL",
+    "BACKGROUND",
+    "LIMITED"
+  }
+end
+
+local function registerApp(pButtonName)
+    common.registerApp()
+    EXPECT_HMICALL("Buttons.SubscribeButton", { appID = common.getHMIAppId(), buttonName = pButtonName })
+    :Do(function(_, data)
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", { })
+    end)
+    EXPECT_HMICALL("Buttons.OnButtonSubscription")
+    :Times(0)
+end
+
+local function registerSoftButton(pAppId)
+    local cid = common.getMobileSession(pAppId):SendRPC("Alert", RequestAlert)
```

Comment:
`local cid =` is not used, please leave only `common.getMobileSession(pAppId):SendRPC("Alert", RequestAlert)`